docs/multicluster/user-guide.md

```diff
@@ -12,6 +12,9 @@
 - [Multi-cluster Service](#multi-cluster-service)
 - [Multi-cluster ClusterNetworkPolicy Replication](#multi-cluster-clusternetworkpolicy-replication)
 - [Multi-cluster Pod to Pod Connectivity](#multi-cluster-pod-to-pod-connectivity)
```

Comment:
Maybe you can move `Multi-cluster Pod to Pod Connectivity` to be before `Multi-cluster ClusterNetworkPolicy Replication` and after `Multi-cluster Service`